security/custom_authenticator.rst

```diff
@@ -176,6 +176,13 @@ can define what happens in these cases:
     above. Use :class:`Symfony\\Component\\Security\\Core\\Exception\\CustomUserMessageAuthenticationException`
     if you want to set custom error messages.
 
+.. tip::
+
+    By default your custom authenticator will not dispatch
+    :class:`Symfony\\Component\\Security\\Http\\Event\\InteractiveLoginEvent`
+    if you need to dispatch this event your authenticator shoud implements
```

Comment:
What about something like:
"If your login method is interactive, which means that the user actively logged into your website,
you may want your authenticator to implement the InteractiveAuthenticatorInterface interface so that it dispatches an InteractiveLoginEvent event"?

Reply:
It's more explicit, thx @mtarld